### DISCUSSION TITLE
Add function to split (dual) tabs like Naver Whale or MS Edge rather than "Glance" feature

### DISCUSSION THREAD

**Discussion by samesdat** [zen-browser/desktop#2995] [Ideas]
Naver Whale Browser (Chromium based) has one extremely useful function (referenced as dual tab) for research, shopping, looking for the right search result in a list, and so on. Description [here](https://help.whale.naver.com/en/desktop/dualtab/) says:

"Dual tab is not simply a split screen. In linked Dual tab, you can click a link on the left tab to open it on the right tab. While using services such as News, Mail, Shopping and Search, you can show the list on the left tab and open the content on the right. You don't have to switch back and forth between tabs."

Once you used it you won't miss it. But the browser Whale itself is ... meh, bloated. 

Someone in Reddit mentioned that there is a Firefox extension which mimics the behavior but I don't know which one.

It would be incredible to add this function. I think it is not difficult ...

**Comment by JSGRD22** [comment]
Edge split screen has a similar feature. It's great for some uses.

**Reply by samesdat** [reply]
Yes. I tried it out at work today and I was surprised how close they copied that feature.  I had to activate ist via round Hamburger menu in the left split window.

So I will change the discussion thread to Naver Whale OR Edge.

Thank you for the good advice!
